.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: php
 
+git:
+  depth: 1
```

Comment:
The [default](https://travis-ci.org/povils/phpmnd/jobs/224318899#L110) is a depth of 50.